---
bool onAir
bool armed
string flightMode